pkg/proxy/iptables/proxier_test.go

```diff
@@ -1345,6 +1347,270 @@ func assertIPTablesRulesNotEqual(t *testing.T, expected, result string) {
 	}
 }
 
+// cidrContains checks if cidrStr contains ipStr. cidrStr can actually be a CIDR or an IP
```

Comment:
Let's make this more explicit

cidrStr is either an IP ("1.2.3.4") or a CIDR ("1.2.3.4/20").